alert/search.go

```diff
@@ -25,11 +25,18 @@ type SearchOptions struct {
 	// Services, if specified, will restrict alerts to those with a matching ServiceID.
 	Services []string `json:"v,omitempty"`
 
+	// AllServices reflects if all services are to return their alerts
+	AllServices bool `json:"l,omitempty"`
+
```

Comment:
This is already accomplished/implied by omitting the `Services` filter.

Reply:
I needed to use this to distinguish between `AllServices` and `FavoritesOnly`, because both trickle down to `.Services` in the query but are handled differently. It's been about a week so I can't remember the exact behavior I was seeing though. I can try to recreate my issue, if you'd like

Reply:
We should work through that a bit more -- they should be equivalent at this level. Either we are filtering by service ID or we're not regardless of why. If we are, then the notified user ID is relevant, otherwise it can be ignored as the service ID filter is the only criteria overridden by the notification ID.

Reply:
Gotcha, I'll circle back to this when I have some time and get back to you